.eslintrc

```diff
@@ -1,12 +1,12 @@
 {
-  "extends": ["airbnb", "plugin:inferno/recommended"],
```

Comment:
Any particular reason why we dropped airbnb? We should probably depend on airbnb without react? https://www.npmjs.com/package/eslint-config-airbnb-base